assembly/README.md

```diff
@@ -47,6 +47,8 @@ begin
 end
 ```
 
+We also provide a concrete implementation of the `Library` trait called `MaslLibrary`. This implementation can be used to instantiate libraries from `.masl` files.
```

Comment:
nit: maybe use `specific` instead of `concrete`?

Reply:
I think `concrete` is fine here.